_scicomputing/compute_jobs.md

```diff
@@ -147,17 +149,22 @@ requested time.
 
 If you should need to increase the amount of time (i.e. increase wall time) for a running job (or jobs), email Scientific Computing.  If a job has not started, you can update this yourself:
 
-    scontrol update jobid=<job ID> timelimit=+2-0
+```
+scontrol update jobid=<job ID> timelimit=+2-0
+```
 
 This command increases the job's timelimit by two days.
 
 ### Why isn't my job running?
 
 There are any number of reasons why your job may not be running.  When you run `squeue` you will see the job's state as `PD` with a reason code (in the column headed with `NODELIST(REASON)`)  As suggested by the heading, this column contains the reason that the job isn't running.
 
-The three most common reasons are a) there are no idle resources for your job, b) jobs running under your account are already consuming the maximum amount of resources available for accounts, and c) your job does not have a high enough priority to be run.
+The three most common reasons are:
+a) there are no idle resources for your job, 
+b) jobs running under your account are already consuming the maximum amount of resources available for accounts, and 
+c) your job does not have a high enough priority to be run.
 
-If your job won't run because resources aren't available, the reason code will be "Resources".  Jobs in this state have resources reserved for them (a "priority reservation") and should run next.  This reason is only applied to one or two jobs in the queue.  Every other job will have the reason "Priority," which indicates that the job does not have sufficient priority to have resources reserved (reason "c" above).
+If your job won't run because resources aren't available, the reason code will be "Resources".  Jobs in this state have resources reserved for them (a "priority reservation") and should run next when those resources become available.  This reason is only applied to one or two jobs in the queue.  Every other job will have the reason "Priority," which indicates that the job does not have sufficient priority to have resources reserved (reason "c" above).
```

Comment:
"This reason is only applied to one or two jobs in the queue."
Is this what you truly mean?  Or do you mean something more like, "Typically, only a small number of jobs should be in the "Resources" state if the jobs are submitted with reasonable resource requests.  A large number of jobs in this state may indicate something is a miss with your request..." Or something like that.